src/main/java/net/atos/zac/signalering/SignaleringenService.java

```diff
@@ -256,4 +270,13 @@ public List<SignaleringInstellingen> listInstellingen(final SignaleringInstellin
     public int count() {
         return SignaleringType.Type.values().length;
     }
+
+    public SignaleringVerzonden createSignaleringVerzonden(final SignaleringVerzonden signaleringVerzonden) {
+        valideerObject(signaleringVerzonden);
+        return entityManager.merge(signaleringVerzonden);
```

Comment:
niet merge maar persist

Reply:
Javadoc: "Merge the state of the given entity into the current persistence context."

Het woord merge verwijst hier niet naar het mergen van velden of zoiets. Ik kan ook wel een persist gebruiken en dan de oorpronkelijke entiteit teruggeven. Maar dit kan ook, zolang ik maar de return teruggeef en niet de oorspronkelijke enititeit want die is na de merge unmanaged.

Persist kun je alleen gebruiken voor nieuwe maar merge kan zowel voor nieuwe als voor het bestaande.